src/test/java/com/checkmarx/flow/cucumber/integration/sca_scanner/bugtrackers/ado/ScaIssuesCreationSteps.java

```diff
@@ -0,0 +1,105 @@
+import java.util.Collections;
+
+@SpringBootTest(classes = {CxFlowApplication.class, AzureDevopsClient.class})
+@RequiredArgsConstructor
+public class ScaIssuesCreationSteps {
+
+    private static final String INPUT_BASE_PATH = "cucumber/data/sample-sca-results/";
+    private final static String NAMESPACE = "CxNamespace";
+    private final static String PROJECT_NAME = "CxScaTest";
+    private static final String AZURE = "Azure";
+    private static final String INPUT_FILE = "8-findings-2-high-6-medium.json";
+
+    private final FlowProperties flowProperties;
+    private final ADOProperties adoProperties;
+    private final AzureDevopsClient azureDevopsClient;
+    private final ResultsService resultsService;
+
+    @Before("@SCA_Issues_Creation")
+    public void init() throws IOException {
+        azureDevopsClient.init(NAMESPACE, PROJECT_NAME);
+        azureDevopsClient.deleteProjectIssues();
+    }
+
+    @After("@SCA_Issues_Creation")
+    public void deleteProjectIssues() throws IOException {
+        azureDevopsClient.deleteProjectIssues();
+    }
+
+    @Given("scan initiator is SCA")
+    public void setScanInitiator() {
+        flowProperties.setEnabledVulnerabilityScanners(Collections.singletonList(ScaProperties.CONFIG_PREFIX));
+    }
+
+    @And("bug tracker is Azure")
+    public void setBugTracker() {
+        flowProperties.setBugTracker(AZURE);
+        flowProperties.setBugTrackerImpl(Collections.singletonList(AZURE));
+        adoProperties.setUrl("https://dev.azure.com/");
+        adoProperties.setProjectName(PROJECT_NAME);
+        adoProperties.setNamespace(NAMESPACE);
+    }
+
+    @When("publishing new known unfiltered SCA results with 8 findings including 2 high and 6 medium vulnerabilities")
+    public void publishNewScaResults() throws IOException, MachinaException {
+        ScanResults scanResults = JsonUtils.json2Object(getFileFromResourcePath(INPUT_BASE_PATH + INPUT_FILE), ScanResults.class);
+        resultsService.processResults(getBasicScanRequest(), scanResults, null);
+    }
+
+    @Then("new {int} tickets should be created")
+    public void ticketsCreationValidation(int expectedTicketsCreation) throws IOException {
+        Assert.assertEquals("Azure new SCA results tickets number is not as expected",
+                expectedTicketsCreation, azureDevopsClient.getIssueCount());
+    }
+
+    private ScanRequest getBasicScanRequest() {
+        return ScanRequest.builder()
+                .product(ScanRequest.Product.CX)
+                .project(PROJECT_NAME)
+                .namespace("cxflowtestuser")
+                .repoName("htmx")
+                .repoType(ScanRequest.Repository.GITHUB)
+                .branch("master")
+                .bugTracker(getAzureCustomBugTracker())
+                .refs(Constants.CX_BRANCH_PREFIX.concat("refs/heads/master"))
+                .build();
+    }
+
+    private File getFileFromResourcePath(String path) throws IOException {
```

Comment:
It's already in com.checkmarx.flow.cucumber.common.utils.TestUtils#getFileFromResource - note that is accepts a **relative** path.

Reply:
The current method in the TestUtils is using a fullpath, while I need a relative one, so I created a new utility method for that case.